Documentation/design-docs/COM-activation.md

```diff
@@ -74,14 +74,12 @@ When `DllGetClassObject()` is called in a COM activation scenario, the following
 1) Based on the `.runtimeconfig.json` the [framework](https://docs.microsoft.com/dotnet/core/packages#frameworks) to use can be determined and the appropriate `hostpolicy` library path is computed.
 1) The `hostpolicy` library is loaded and various exports are retrieved.
     * If a `hostpolicy` instance is already loaded, the one presently loaded is re-used.
-    * **Prior to 3.0 GA** No validation is done to determine if the loaded `hostpolicy` can satisfy the current assembly's `.runtimeconfig.json`.
-    * **At 3.0 GA** If a CLR is active within the process, the requested CLR version will be validated against that CLR. If version satisfiability fails, activation will fail.
+    * If a CLR is active within the process, the requested CLR version will be validated against that CLR. If version satisfiability fails, activation will fail.
```

Comment:
Do we care about pointing out the bit of self-contained sadness (at least in 3.0) where we don't do this validation?

Reply:
No at present. I am deferring that to general issues with self-contained scenarios. I say that because COM is impacted, but so are all other host related scenarios so would prefer to keep this entirely about COM activation.